_inc/client/components/jumpstart/style.scss

```diff
@@ -33,4 +33,10 @@
 			margin-top: 0;
 		}
 	}
+
+	.jp-jumpstart__skip-link {
```

Comment:
Does this need to be nested?

Reply:
Yes, because the `.is-borderless` class has a higher specificity IIRC, and no margins. If we remove the class as you suggest below, then we can move it down a level.

Reply:
Removed the classes from the button, and this CSS from the stylesheet.